src/table/hooks/use-field-selection.ts

```diff
@@ -30,7 +40,18 @@ const useFieldSelection = (column: Column): UseFieldSelectionOutput => {
 
   useEffect(() => {
     if (!app || !app.getField || !column || !column.isDim) return;
-    app.getField(column.fieldId, layout.qStateName).then(setFieldInstance);
+    const { qLibraryId, fieldId } = column;
+    qLibraryId
```

Comment:
This would probably be easier to read if it was a "if else statement" rather then a ternary operator